src/org/labkey/test/params/experiment/SampleTypeDefinition.java

```diff
@@ -68,6 +71,28 @@ public SampleTypeDefinition setDescription(String description)
         return this;
     }
 
+    protected MetricUnit getInventoryMetricUnit()
+    {
+        return _inventoryMetricUnit;
+    }
+
+    protected SampleTypeDefinition setInventoryMetricUnit(MetricUnit inventoryMetricUnit)
+    {
+        _inventoryMetricUnit = inventoryMetricUnit;
+        return this;
+    }
+
+    protected String getLabelColor()
+    {
+        return _labelColor;
+    }
+
+    protected SampleTypeDefinition setLabelColor(String color)
+    {
+        _labelColor = color;
+        return this;
+    }
+
```

Comment:
To make these work with API helpers, add `metricUnit` and `labelColor` to the map produced by `getOptions`:
```
if (getInventoryMetricUnit() != null)
{
    options.put("metricUnit", getInventoryMetricUnit());
}
if (getLabelColor() != null)
{
    options.put("labelColor", getLabelColor());
}
```

Reply:
Updated.